public/scripts/loginController.js

```diff
@@ -0,0 +1,37 @@
+(function(module) {
+  var loginController = {};
+
+  loginController.index = function() {
+    $('#login').show().siblings().hide();
+    $('.css-home-login-link').hide();
+    $('#mainNav').hide();
+    loginController.verifyuser();
+  };
+
+  loginController.verifyuser = function() {
+
+    $('#login-page').on('submit', function(e) {
+
+      e.preventDefault();
+      $.get('/data', {email: e.target.email.value}).done(function(result) {
+
+        if (result.rows.length === 0) {
+          alert('database says you do not exist');
```

Comment:
this was the one piece of UI that seemed out of step with the tone of the app. it's much less friendly than everything else.